package/yast2-network.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Nov 25 08:34:55 UTC 2019 - Knut Anderssen <kanderssen@suse.com>
+
+- When reading udev rules, skip from the assigned udev rule parts
```

Comment:
np: I would simplify this sentence: "Ignores invalid udev rules parts." or something like that.